File: python/practices/mad_libs.py
# AP Mad Libs
adjective = input("can you give me a adjective? ").strip()
noun = input("can you give me a animal? ").strip().capitalize()
verb = input("can you give me a verb in past tense? ").strip()
adverb = input("can you give me a adverb? ").strip()
another_AD = input("can you give me another adjective? ").strip()
another_noun = input("can you give me another noun? ").strip().capitalize()

sentence = "Today I went to the zoo. I saw a(n) " + adjective + " " + noun + " jumping up and down in its tree. He " + adverb + " " + verb + " through the large tunnel that led to its " + another_AD + " " + another_noun + "."
print(sentence)

File: C/practices/mad_libs.c
//AP silliy sentences
#include <stdio.h> 
#include <string.h>
int main(){
    char adjective[100];
    char noun[100];
    char adverb[100];
    char another_AD[100];
    char another_noun[100];
    char verb[100];
    char space[] = " ";

    strcat(adjective, space);
    strcat(adverb, space);
    strcat(another_AD, space);

    printf("can you give me a adjective? \n");
    scanf("%s", &adjective);
    
    printf("can you give me a noun?\n");
    scanf("%s", &noun);
     
    printf("can you give me a verb in past tense?\n");
    scanf("%s", &verb);
    
    printf("can you give me a adverb?\n");
    scanf("%s", &adverb);
    
    printf("can you give me another adjective?\n");
    scanf("%s", &another_AD);
    
    printf("can you give me another noun?\n");
    scanf("%s", &another_noun);


    






}